test/integration/test_assistant_v2.rb

```diff
@@ -12,7 +12,8 @@ class AssistantV2Test < Minitest::Test
     attr_accessor :service
     def before_all
       authenticator = IBMWatson::Authenticators::IamAuthenticator.new(
-        apikey: ENV["ASSISTANT_APIKEY"]
+        apikey: ENV["ASSISTANT_APIKEY"],
+        url: ENV["ASSISTANT_AUTH_URL"]
```

Comment:
will this work in ci? do we have auth url defined?

Reply:
dont know why its there but it does seem to work.  I am going to try and remove it.